tests/test_ssl.py

```diff
@@ -105,6 +105,9 @@ def remove_files():
 
 def test_run(create_certfile_and_keyfile):
 
+    if sys.platform.startswith("win"):
+        pytest.skip("Skipping SSL test on Windows for now :(")
```

Comment:
Because they hang. Have not investigated further, but I don't want it to hold back this PR.

Reply:
That's fair enough, yup!